tests/filesystem.go

```diff
@@ -187,8 +186,11 @@ func setDevices(t *testing.T, ctx context.Context, imageFile string, partitions
 	}
 	loopDevice := strings.TrimSpace(string(out))
 
-	// To avoid races with the kernel with loop devices appearing, sleep a bit
-	time.Sleep(time.Millisecond * 100)
+	// Avoid race with kernel by waiting for loopDevice creation to complete
+	_, err = run(t, ctx, "udevadm", "settle")
+	if err != nil {
+		return "", fmt.Errorf("Settling devices: %v type", err)
```

Comment:
Our typical style is to end with the `%v`

Reply:
I don't remember typing "type" in there.  I wonder if I was put in while playing around with my new IDE.  Anyhow, fixed it up.